website/docs/plugin/framework/migrating/schema.mdx

```diff
@@ -0,0 +1,142 @@
+---
+page_title: 'Provider: Migrating from SDKv2 to the Framework'
+description: >-
+   Migrate a schema from SDKv2 to the plugin Framework.
+---
+
+# Schema
```

Comment:
Happy to do this. @robin-norwood @laurapacilio can you confirm that we're all aligned on this before I make the relevant changes. Thanks.

Reply:
I'm confused by this because I thought schemas were common to all types --> provider, data source, etc. Is the content on this page only relevant to attributes?

Reply:
Have promoted this page so it appears as "Schema - Overview" and added the links you supplied in your suggested change.